packages/breadcrumbs/index.js

```diff
@@ -1,20 +1,25 @@
-import { FabricWebComponent } from '../utils';
+import { html } from 'lit';
+import { FabricElement } from '../utils';
 
-class FabricBreadcrumbs extends FabricWebComponent {
-  connectedCallback() {
-    const children = Array.from(this.children)
-      .map((child) => child.outerHTML)
-      .join('<span class="select-none" aria-hidden="true">/</span>');
+const separator = html`<span class="select-none" aria-hidden="true">/</span>`;
+const interleave = (arr) =>
+  [].concat(...arr.map((el) => [el, separator])).slice(0, -1);
```

Comment:
Whats the purpose of the `[].concat` here?

Reply:
map creates pairs of elements and separators `[el, separator] [el, separator] [el, separator]`, concat joins these array pairs together so you end up with 1 array with `[el, separator, el, separator, el, separator]`. Then the slice trims off the last separator so you end up with `[el, separator, el, separator, el]`

Reply:
If we don't have to support Edge 16-18 or iOS11 we could instead `array.flatMap(el => [el, separator]).slice(1)`